src/site/xdoc/server/config-dnsservice.xml

```diff
@@ -61,8 +61,8 @@
            This should always be <strong>false</strong> unless you understand the implications.</dd>
         <dt><strong>maxcachesize</strong></dt>
         <dd>Maximum number of entries to maintain in the DNS cache (typically 50000)</dd>
-        <dt><strong>cacheTTL</strong></dt>
-        <dd>Sets the maximum length of time that records will be stored in the DNS cache in seconds. A negative value disables this feature (that is, sets no limit).</dd>
+        <dt><strong>negativeCacheTTL</strong></dt>
+        <dd>Sets the maximum length of time that negative records will be stored in the DNS negative cache in seconds. A negative value disables this feature (that is, sets no limit).</dd>
```

Comment:
I would say that a negative value activate infinitely this feature, while a zero value deactivate it, no?